lively.collab/comments/commentBrowser.js

```diff
@@ -184,8 +186,21 @@ export class CommentBrowser extends Window {
 
     containerToRemove.remove();
     this.container.addMorph(newContainer);
-    containerToRemove.submorphs.forEach((commentGroup) => commentGroup.hideCommentIndicators());
-    newContainer.submorphs.forEach((commentGroup) => commentGroup.showCommentIndicators());
+    this.hideAllCommentIndicators();
+    this.showCommentIndicators();
+  }
+
+  showCommentIndicators () {
+    if (this.showsResolvedComments) {
+      this.resolvedCommentContainer.submorphs.forEach((commentGroup) => commentGroup.showCommentIndicators());
```

Comment:
Please use `withAllSubmorphsDo` to iterate over submoprhs.

Reply:
Please don't

Reply:
we don't want to do it with all submorphs but only with the first hierarchy line

Reply:
.